<?xml version="1.0"?>
<launch>
    <arg name="hostname" default="192.168.0.1"/>
    <node name="sick_nav_2xx" pkg="sick_scan" type="sick_generic_caller" respawn="false" output="screen">
        <param name="intensity" type="bool" value="true"/>
        <param name="intensity_resolution_16bit" type="bool" value="true"/>
        <param name="min_ang" type="double" value="-2.35619"/>
        <param name="max_ang" type="double" value="2.35619"/>
        <param name="frame_id" type="str" value="laser"/>
        <param name="use_binary_protocol" type="bool" value="true"/>
        <param name="scanner_type" type="string" value="sick_nav_2xx"/>
        <param name="hostname" type="string" value="$(arg hostname)"/>
        <param name="port" type="string" value="2112"/>
        <param name="timelimit" type="int" value="5"/>
    </node>
</launch>
